src/lua/init.c

```diff
@@ -793,6 +806,32 @@ luaopen_tarantool(lua_State *L)
 	return 1;
 }
 
+void
+tarantool_lua_load_modules(lua_State *L, const char * const *modules)
```

Comment:
Moving code to a separate function fails on checkpatch with `ERROR:STATIC_CONST_CHAR_ARRAY: static const char * array should probably be static const char * const` and style errors (line too long, indentation is bad).